webapp/src/errorList.tsx

```diff
@@ -108,4 +113,16 @@ export class ErrorList extends React.Component<ErrorListProps, ErrorListState> {
             exception
         })
     }
+
+    private generateStackTraces(exception: pxsim.DebuggerBreakpointMessage) {
+        return (
+            <div>
+                {(exception.stackframes).map(sf =>
+                    <div className="stackframe">
```

Comment:
Clickeable?

Reply:
Yeah I'm going to add that in the next pr